logging/src/main/java/com/virtuslab/logger/PrefixedLambdaLogger.java

```diff
@@ -60,38 +60,18 @@ public void info(String format) {
     logger.info(() -> getLogMessagePrefix() + format);
   }
 
-  @Override
-  public void info(Supplier<String> msgSupplier) {
```

Comment:
@micpiotrowski to confirm, but I don't think there's any benefit of having lazy log string evaluation for levels above DEBUG; coz IIUC IntelliJ sets the default level to INFO for all topic anyway

Reply:
But it also doesn't bother us and maybe in future default loglevel in IntelliJ will be changed.
But ok, it also can be deleted: use your judgement.

Reply:
Restored by yourself on #242, let's consider it resolved then